app/client/stylesheets/_login.scss

```diff
@@ -9,10 +9,28 @@
 #login {
   height: 100%;
   width: 100%;
-  background-color: $login-bg-color;
-  background: $login-bg;
-  background-size: cover;
-  overflow: hidden;
+
+  > img.background {
+    height: 100%;
+    width: 101%;
+    overflow: hidden;
+    z-index: -1;
+    position: absolute;
+    object-fit: cover;
+
+    &.b-lazy{
+      /* This will harden up the edges on the blur*/
+      width: 102%;
```

Comment:
I'm really confused about these percentages and margins? Are they specific to Blazy?

Reply:
No but the blur effect causes the edges to turn this ugly white and it looks awful. You can remove them and see it for yourself, this keeps the edges hard for the blurred image

Reply:
Okay, I see it now. Can we maybe do:
```
width: calc(100% + 16px);
height: calc(100% + 16px);
margin: -16px;
```